docker/tidb-bench/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.12.1-alpine3.9 AS goycsb
+
+RUN apk add git build-base \
+  && git clone https://github.com/pingcap/go-ycsb $GOPATH/src/github.com/pingcap/go-ycsb \
+  && cd $GOPATH/src/github.com/pingcap/go-ycsb \
+  && make
+
+FROM alpine:3.9
+MAINTAINER PingCAP <cloud@pingcap.com>
+
+COPY --from=pingcap/tidb-sysbench /usr/local/bin/sysbench /usr/local/bin/sysbench
+COPY --from=pingcap/tidb-sysbench /usr/local/share/sysbench /usr/local/share/sysbench
+
+COPY --from=dmonakhov/alpine-fio /usr/local/bin/fio /usr/local/bin/fio
+
+COPY --from=goycsb /go/src/github.com/pingcap/go-ycsb/bin/go-ycsb /go-ycsb/bin/go-ycsb
```

Comment:
Also does this work in alpine? And maybe you can copy from pingcap/go-ycsb directly instead of build the go-ycsb yourself.

Reply:
I haven't tested this, but it should work. Copying from pingcap/go-ycsb creates additional fragility unless we define a contract for go-ycsb (linux distro, binary location, static build)

Reply:
The go-ycsb repo already has a [Dockerfile](https://github.com/pingcap/go-ycsb/blob/master/Dockerfile). It also uses alpine as builder and final image.